packages/projects-docs/pages/learn/editors/vscode/_meta.json

```diff
@@ -0,0 +1,4 @@
+{
+  "overview": "Overview",
+  "setting-up-vscode": "VSCode Setup"
```

Comment:
This should be `"vs-code-setup": "VS Code Setup"`
This helps ensure consistency between title and slug.